lib/node_modules/@stdlib/stats/base/dmeankbn/manifest.json

```diff
@@ -27,18 +28,19 @@
   "confs": [
     {
       "task": "build",
+      "wasm": false,
       "src": [
-        "./src/dmeankbn.c"
+        "./src/main.c"
       ],
       "include": [
         "./include"
       ],
-      "libraries": [
-        "-lm"
-      ],
+      "libraries": [],
       "libpath": [],
       "dependencies": [
-        "@stdlib/blas/ext/base/dsum",
+        "@stdlib/blas/base/shared",
+        "@stdlib/strided/base/stride2offset",
+        "@stdlib/blas/ext/base/dsumkbn",
```

Comment:
@aayush0325 Thanks for catching this. Yes, should have been `dsumkbn`, not `dsum`.